They are incompatible with musl's iconv implementation relying on
encodings that are not supported by musl libc et cetera.

diff -upr csv9.6.4.orig/mats/io.ms csv9.6.4/mats/io.ms
--- csv9.6.4.orig/mats/io.ms	2023-10-22 22:37:14.797906883 +0200
+++ csv9.6.4/mats/io.ms	2023-10-22 22:40:02.128273023 +0200
@@ -534,7 +534,8 @@
      1)
  )
 
-(if (embedded?)
+;; Disable iconv tests on musl due to shortcoming of its iconv implementation.
+(if #t
     (mat iconv-codec
       (error? (errorf 'iconv-codec "-73 is not a string"))
       (error? (errorf 'transcoded-port "unsupported encoding almost certainly bogus"))
diff -upr csv9.6.4.orig/mats/root-experr-compile-0-f-f-f csv9.6.4/mats/root-experr-compile-0-f-f-f
--- csv9.6.4.orig/mats/root-experr-compile-0-f-f-f	2023-10-22 22:37:14.797906883 +0200
+++ csv9.6.4/mats/root-experr-compile-0-f-f-f	2023-10-22 22:39:56.678263008 +0200
@@ -6871,21 +6871,6 @@ io.mo:Expected error in mat port-operati
 io.mo:Expected error in mat port-operations3: "port-file-descriptor: #<input port string> is not a file port".
 io.mo:Expected error in mat iconv-codec: "iconv-codec: -73 is not a string".
 io.mo:Expected error in mat iconv-codec: "transcoded-port: unsupported encoding almost certainly bogus".
-io.mo:Expected error in mat iconv-codec: "display-string: iconv CP1252 codec cannot encode #\x3BB".
-io.mo:Expected error in mat iconv-codec: "newline: iconv CP1252 codec cannot encode #\newline with eol-style ls".
-io.mo:Expected error in mat iconv-codec: "newline: latin-1 codec cannot encode #\newline with eol-style ls".
-io.mo:Expected error in mat iconv-codec: "iconv-codec: -73 is not a string".
-io.mo:Expected error in mat iconv-codec: "transcoded-port: unsupported encoding almost certainly bogus".
-io.mo:Expected error in mat iconv-codec: "display-string: iconv CP1252 codec cannot encode #\x3BB".
-io.mo:Expected error in mat iconv-codec: "newline: iconv CP1252 codec cannot encode #\newline with eol-style ls".
-io.mo:Expected error in mat iconv-codec: "newline: latin-1 codec cannot encode #\newline with eol-style ls".
-io.mo:Expected error in mat iconv-codec: "iconv-codec: -73 is not a string".
-io.mo:Expected error in mat iconv-codec: "transcoded-port: unsupported encoding almost certainly bogus".
-io.mo:Expected error in mat iconv-codec: "close-port: iconv CP1252 codec cannot encode #\x3BB".
-io.mo:Expected error in mat iconv-codec: "close-port: iconv CP1252 codec cannot encode #\newline with eol-style ls".
-io.mo:Expected error in mat iconv-codec: "close-port: latin-1 codec cannot encode #\newline with eol-style ls".
-io.mo:Expected error in mat iconv-codec: "iconv-codec: -73 is not a string".
-io.mo:Expected error in mat iconv-codec: "transcoded-port: unsupported encoding almost certainly bogus".
 io.mo:Expected error in mat iconv-codec: "close-port: iconv CP1252 codec cannot encode #\x3BB".
 io.mo:Expected error in mat iconv-codec: "close-port: iconv CP1252 codec cannot encode #\newline with eol-style ls".
 io.mo:Expected error in mat iconv-codec: "close-port: latin-1 codec cannot encode #\newline with eol-style ls".
